src/jshint.js

```diff
@@ -3008,6 +3027,7 @@ var JSHINT = (function() {
         if (left.value.match(/^[A-Z]([A-Z0-9_$]*[a-z][A-Za-z0-9_$]*)?$/)) {
           if ("Array Number String Boolean Date Object Error Symbol".indexOf(left.value) === -1) {
             if (left.value === "Math") {
+              /* istanbul ignore next */
               warning("W063", left);
```

Comment:
@jugglinmike I suggest that we update "W063" to be `"{a} is not a function."` and add `left.value` as a third parameter here. Then we can catch Atomics, JSON and Reflect